.github/workflows/sdk-examples.yaml

```diff
@@ -40,6 +40,7 @@ jobs:
         run: |
           set -euo pipefail
           apt update
+          DEBIAN_FRONTEND=noninteractive TZ=UTC apt install -y -q libprotobuf-dev libopencv-dev
```

Comment:
I would also add a comment here explaining why this exist. Else someone in the future will see it and be like.. Why? I should remove it (... proceed to see CI blowing up).